### DISCUSSION TITLE
Documentation incomplete

### DISCUSSION THREAD

**Discussion by krisavi** [cybertec-postgresql/pgwatch#585] [Q&A]
**Describe the bug**
Documentation is very lacking. ['Custom installation'](https://github.com/cybertec-postgresql/pgwatch/blob/master/docs/tutorial/custom_installation.md) starts with building Go applications and then there are references to files that do not exist and after that it goes to instruct using files and parameters that were used for PgWatch2. Neither code nor --help doesn't reveal what parameters should be used to set up non-docker installations.

**To Reproduce**
1. Go to ['Custom installation'](https://github.com/cybertec-postgresql/pgwatch/blob/master/docs/tutorial/custom_installation.md)
2. Try to follow or set up using it

**Expected behavior**
Documentation to have correct information.

**Additional context**

> Sample startup scripts can be found at /etc/pgwatch/startup-scripts/pgwatch.service or online [here](https://github.com/cybertec-postgresql/pgwatch/blob/master/pgwatch/startup-scripts/pgwatch.service). Note that they are OS agnostic and might need some light adjustment of paths, etc - so always test them out.

Nope, nope, nope. Just doesn't exist in those locations.

**Comment by ondar** [comment]
Yeah, I went through the docs too and for a lot of typos and remnants from pgwatch2. See https://github.com/cybertec-postgresql/pgwatch/pull/570

**Comment by pashagolub** [comment]
Yes, this is true, unfortunately. I'm trying to find some time to work on it but it's always with lowest priority :(

**Comment by krisavi** [comment]
~Been trying to figure out how to configure patroni lookup for the database, but don't see any way to configure etcd parameters.~
For database it asks for connectionstring, entering password plaintext into there doesn't seem like a good idea either.

Oh, found host config... The column was hidden by default, so have to turn it on to be able to modify it and add DCS info.

**Comment by philippe-vs** [comment]
Hi folks,

Will it exist a migration path from pgwatch2 to pgwatch3 documented in the documentation ?

Thanks in advance.

**Reply by pashagolub** [reply]
Hi, yes on my todo list! :-) As well as some workflow. Right now if you want to convert **custom** metrics, there is a [`convert_metrics` utility](https://github.com/cybertec-postgresql/pgwatch/tree/master/cmd/convert_metrics). Measurements database should work as it is now.

The only problem might be with converting config database. Nothing extraordinary anyway.

**Comment by pashagolub** [comment]
Docs are up to date now. Please, create a separate issue if you will find any inconsistency. Thanks for your help!
